CoEpi/ui/onboarding/AttributedString.swift

```diff
@@ -0,0 +1,71 @@
+import UIKit
+
+typealias FontAttributes = [NSAttributedString.Key: Any]
+
+struct AttributableString {
+    
+    let baseFont: UIFont
+    var storage = NSMutableAttributedString()
+    
+    let boldAttributes: FontAttributes
+    let normalAttributes: FontAttributes
+
+    init(baseFont: UIFont) {
+        
+        self.baseFont = baseFont
+        
+        let pointSize = baseFont.pointSize
+        let fontDescriptor = baseFont.fontDescriptor.withSymbolicTraits(.traitBold)
+            ?? UIFont.boldSystemFont(ofSize: pointSize).fontDescriptor
+        
+        let boldFont = UIFont(descriptor: fontDescriptor, size: pointSize)
+        
+        self.boldAttributes = [
+            .font: boldFont
+        ]
+        
+        self.normalAttributes = [
+            .font: baseFont
+        ]
+        
+    }
+    
+    func append(_ value: String, attributes: FontAttributes) {
+        let attributedString = NSAttributedString(string: "\(value) ", attributes: attributes)
```

Comment:
I'd expect this function to just append `value` and not prepend a space to it. Maybe create a new function `space` that inserts the space and call it explicitly when building the string.